pkg/provider/data_install.go

```diff
@@ -92,10 +93,11 @@ func DataInstall() *schema.Resource {
 				Default:     installDefaults.NetworkPolicy,
 			},
 			"log_level": {
-				Description: "Log level for toolkit components.",
-				Type:        schema.TypeString,
-				Optional:    true,
-				Default:     installDefaults.LogLevel,
+				Description:  "Log level for toolkit components.",
+				Type:         schema.TypeString,
+				Optional:     true,
+				Default:      installDefaults.LogLevel,
+				ValidateFunc: validation.StringInSlice([]string{"info", "debug", "error"}, false),
```

Comment:
Is it possible to get these values from a package in flux2?

Reply:
No, but we could expose the args accepted values in a package. I would merge the validation and refactor later.

Reply:
I tried that but they are not exposed as a pkg so I couldn't get at them, maybe a more experienced gopher could.

Reply:
I will create an issue to track this so we can merge this first